src/data_conn.c

```diff
@@ -247,6 +250,7 @@ handle_epoll_out_event(replica_t *r)
 			return -1;
 		if (ret == WRITE_COMPLETED) {
 			TAILQ_REMOVE(&r->readyq, cmd, next);
+			clock_gettime(CLOCK_MONOTONIC, &cmd->queued_time);
```

Comment:
let us set the queued time from the point when it gets added to blocked q or ready q

Reply:
ok